mlir/lib/Transforms/Utils/DialectConversion.cpp

```diff
@@ -1538,19 +1538,6 @@ ConversionPatternRewriter::ConversionPatternRewriter(MLIRContext *ctx)
 
 ConversionPatternRewriter::~ConversionPatternRewriter() = default;
 
-void ConversionPatternRewriter::replaceOpWithIf(
-    Operation *op, ValueRange newValues, bool *allUsesReplaced,
-    llvm::unique_function<bool(OpOperand &) const> functor) {
-  // TODO: To support this we will need to rework a bit of how replacements are
```

Comment:
Note: `replaceAllUsesWith` / `replaceUsesWithIf` is not supported by the dialect conversion either. I have a prototype that adds support for `replaceAllUsesWith`. At that point, I will  override `replaceUsesWithIf` in the same way as `replaceOpWithIf` is at the moment (and put an `llvm_unrechable`).